.github/settings.yml

```diff
@@ -0,0 +1,38 @@
+# These settings are synced to GitHub by https://probot.github.io/apps/settings/
+
+branches:
+  - name: master
+    # https://docs.github.com/en/rest/reference/repos#update-branch-protection
+    # Branch Protection settings. Set to null to disable
+    protection:
+      # Required. Require at least one approving review on a pull request, before merging. Set to null to disable.
+      required_pull_request_reviews:
+        # The number of approvals required. (1-6)
+        required_approving_review_count: 1
+        # Dismiss approved reviews automatically when a new commit is pushed.
+        dismiss_stale_reviews: true
+        # Blocks merge until code owners have reviewed.
+        require_code_owner_reviews: true
+        # Specify which users and teams can dismiss pull request reviews. Pass an empty dismissal_restrictions object to disable. User and team dismissal_restrictions are only available for organization-owned repositories. Omit this parameter for personal repositories.
+        dismissal_restrictions:
+          users: []
+          teams: 
+            - platform-console-services
+      # Required. Require status checks to pass before merging. Set to null to disable
+      required_status_checks:
+        # Required. Require branches to be up to date before merging.
+        strict: true
+        # Required. The list of status checks to require in order to merge into this branch
+        contexts: 
+          - Test
+          - Linting and Security
+          - danger/danger
+      # Required. Enforce all configured restrictions for administrators. Set to true to enforce required status checks for repository administrators. Set to null to disable.
+      enforce_admins: true
+      # Prevent merge commits from being pushed to matching branches
+      required_linear_history: true
+      # Required. Restrict who can push to this branch. Team and user restrictions are only available for organization-owned repositories. Set to null to disable.
+      restrictions:
+        apps: null
+        users: null
+        teams: null
```

Comment:
I'm comparing the settings in this file to the original PR: https://github.com/department-of-veterans-affairs/vets-api/pull/12631 and I see the `restrictions` are updated to `null`. 

When the original PR merged, it seemed like there were TWO settings that were wrong.
> The two settings I unchecked in the UI were:
> 1. “Restrict who can push to matching branches”
> 2. “Require branches to be up to date before merging”

Holden found that setting these values to `null` would fix number 1. @rjohnson2011 What will fix number 2? We always give the _option_ of updating a branch, but we don't want to make that _required_.

Reply:
@rmtolmach I updated the branch to make this `strict: false` (not requiring branches to be up to date before merging) as according to these docs -> https://registry.terraform.io/providers/integrations/github/latest/docs/resources/branch_protection

Thanks for catching that.